intro-to-R-tidyverse/README.md

```diff
@@ -0,0 +1,15 @@
+# Intro to R and Tidyverse Training Module
+
+This CCDL-designed module covers the introduction to R and the Tidyverse package. It is designed to be taught in 1.5 hours. 
+
+It goes through the basics of R _(including data types, data structures, and good scientific coding practices)_ and the tidyverse package _(including dplyr pipes, common tidyverse functions, and the usage of ggplot2 for data visualization)_, thus providing the tools needed to move onto the subsequent training modules. 
+
+The notebooks that comprise this module are: 
+
+* [A quick guide to RStudio](https://github.com/AlexsLemonade/training-modules/blob/master/intro-to-R-tidyverse/00a-rstudio_guide.md) 
```

Comment:
Remove any reference to the `00` notebooks from this list to be consistent with the RNA-seq module README.